### PR TITLE
Add target_file flag to semgrep-core

### PR DESCRIPTION
**Pull request by brendongo**
Allows us to pass a newline delimited list of files to semgrep-core
instead of using the cli so we can get around shell character limits

### CODE REVIEW

**Comment by brendongo** [comment]
Closes #911

File: semgrep-core/bin/Main.ml [hunk 1]
```diff
@@ -878,7 +882,14 @@ let main () =
   in
 
   (* does side effect on many global flags *)
-  let args = Common.parse_options (options()) usage_msg (Array.of_list argv) in
+  let _args_tmp = Common.parse_options (options()) usage_msg (Array.of_list argv) in
```
Comment (aryx) [inline semgrep-core/bin/Main.ml]: Do not use _ as the first chat. It is used to silence warnings about unused variable. Here the var will be used

Comment (aryx) [inline semgrep-core/bin/Main.ml]: You can call it args. I ocaml you can do let x = a in let x = if t then x else b in

**Review by aryx — APPROVED** [review]

File: semgrep-core/bin/Main.ml (hunk 1 above)
Comment (brendongo) [reply semgrep-core/bin/Main.ml]: Went with this suggestion!

**Pull request merged**